City-of-Helsinki/drupal-helfi-kymp
# UHF-2828: matomo settings

How to test:

- Install KYMP instance
- Checkout this branch: `git checkout UHF-2828_matomo_settings`
- Import config: `make drush-cim`
- See that
  - `helfi_matomo_config`, `matomo` and `matomo_reports` modules have been enabled
  - both Matomo modules have config in place
  - permissions for admin are the same as in `conf/cmi/user.role.admin.yml`
